htdocs/misc/map.image.php

```diff
@@ -259,45 +260,45 @@
 
         if (($admin && !isset($user)) || $user->discovered((($arr['sx'] - 1) * $cx_num) + $arr['cx'],(($arr['sy'] - 1) * $cy_num) + $arr['cy']))
         {
-            if ($arr['code']=='s')
+            if ($arr['code']==EntityType::STAR)
             {
-            $sres = dbquery("
-            SELECT
-                type_id
-            FROM
-                stars
-            WHERE
-                id=".$arr['id']."
-            LIMIT 1;
-            ");
-            $sarr = mysql_fetch_row($sres);
-            $starImageSrc = imagecreatefrompng(IMG_DIR."/stars/star".$sarr[0]."_small.png");
-            imagecopyresampled($im,$starImageSrc,$xe,$ye,0,0,GALAXY_IMAGE_SCALE,GALAXY_IMAGE_SCALE,imagesx($starImageSrc),imagesy($starImageSrc));
+                $sres = dbquery("
```

Comment:
you could use the StarRepository here

Reply:
Good point, haven't really looked at this file so far

Reply:
Done